starboard/nplb/directory_can_open_test.cc

```diff
@@ -34,6 +35,11 @@ TEST(SbDirectoryCanOpenTest, SunnyDay) {
 }
 
 TEST(SbDirectoryCanOpenTest, SunnyDayStaticContent) {
+  // TODO, b/380344836, the test cases are not applicable on Android
+  if (GetRuntimePlatform() == PlatformType::kPlatformTypeAndroid) {
+    GTEST_SKIP() << "Not applicable on Android";
```

Comment:
With this and subsequent SKIPs, could we remove some suppressions from [test_filters.py](https://source.corp.google.com/h/lbshell-internal/cobalt_src/+/main:starboard/android/shared/test_filters.py;l=54-104) ?